docs/Hardlinks/Hardlinks-and-Instant-Moves.md

```diff
@@ -36,30 +36,34 @@ Then Continue to [How to set up for](/Hardlinks/How-to-setup-for/) your installa
 
         This means:
 
-            - You can delete any "copy" w/o impacting the others.
+        - You can delete any "copy" without impacting other instances (hardlinks) of it.
 
-                Your download client can remove it's "copy" w/o impacting the library "copy". The library "copy" can be removed by Plex or Sonarr/Radarr or yourself w/o impacting the download client "copy".
+            Your download client can remove its "copy" without impacting the library "copy". The library "copy" can also be removed by Plex, Sonarr/Radarr, or manually without impacting your download client's "copy".
 
-            - Space is only regained when all "copies" are deleted.
+        - Space is only reclaimed once all "copies" of data that are hardlinked are deleted.
 
-            - Modifying the file will impact all "copies".
+        - Modifying any copy of a hardlinked file will impact all "copies".
 
-                For example, modifying the id3 tags of a .mp3 download after import would modify the download client "copy", resulting in breaking the torrent.
+            For example, modifying the id3 tags of a .mp3 download after import would modify the download client "copy", resulting in breaking the torrent.
 
         [More info from Wikipedia, the free encyclopedia](https://en.wikipedia.org/wiki/Hard_link){:target="_blank" rel="noopener noreferrer"}
 
-        !!! warning
-
-            You can't create hard links for directories :bangbang:
-
-            You can't cross file systems :bangbang:
+    !!! warning "Don't forget to read the [Hardlinks limitations](/Hardlinks/Hardlinks-and-Instant-Moves/#hardlinks-limitations)"
 
 ### What are Instant Moves (Atomic Moves)
 
 ??? question "**What are Instant Moves (Atomic Moves)?** - [Click to show/hide]"
 
     A real move and not a copy file from download folder to media folder and then delete file from download folder.
 
+### Hardlinks Limitations
+
+??? question "**Hardlinks limitations** - [Click to show/hide]"
+
+    - You **CAN'T** create hard links for directories :bangbang:
+    - You **CAN'T**  hardlink across separate file systems, partitions, or mounts :bangbang:
+    - Some file systems, such as exFAT, are known not to support hard links and should be avoided (double-check if you are unsure!)
```

Comment:
are we going with `hardlinks` or `hard links`?

Reply:
I believe both grammatically and technically it is supposed to be `hard link` - but this is kinda irrelevant. Pick one, keep it consistent. It's not unclear either way...just potentially confusing to switch back and forth.

Reply:
Fixed with: https://github.com/TRaSH-Guides/Guides/pull/2004/commits/07ceefa9847776ba4e1f714900cf7b683ab367ee